paddle/fluid/platform/gpu_info.cc

```diff
@@ -73,6 +74,10 @@ int GetCUDADeviceCount() {
   return dev_cnt;
 }
 
+/* Here is a very simple CUDA “pro tip”: cudaDeviceGetAttribute() is a much
```

Comment:
why is this comment here?  looks like that `cudaDeviceGetAttribute` is not used anyway

Reply:
This comment is for GetCUDAComputeCapability, I change it to right position by the way